File: dotnet/Models/dto/Threeds1/ThreeDS1AuthResp.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace GPayments.Requestor.TestLab.Models.dto.Threeds1
{
    public class ThreeDS1AuthResp
    {
        public String threeDSRequestorTransID;
        public String challengeUrl;

        public override string ToString()
        {
            return "ThreeDS1AuthResp{" +
                "challengeUrl='" + challengeUrl + '\'' +
                '}';
        }
    }
}

File: java/src/main/java/com/gpayments/requestor/testlab/dto/threeds1/ThreeDS1AuthResp.java
package com.gpayments.requestor.testlab.dto.threeds1;

public class ThreeDS1AuthResp {


  private String threeDSRequestorTransID;
  private String challengeUrl;


  public String getThreeDSRequestorTransID() {
    return threeDSRequestorTransID;
  }

  public void setThreeDSRequestorTransID(String threeDSRequestorTransID) {
    this.threeDSRequestorTransID = threeDSRequestorTransID;
  }

  public String getChallengeUrl() {
    return challengeUrl;
  }

  public void setChallengeUrl(String challengeUrl) {
    this.challengeUrl = challengeUrl;
  }

  @Override
  public String toString() {
    return "ThreeDS1AuthResp{" +
        "challengeUrl='" + challengeUrl + '\'' +
        '}';
  }
}
